spring-integration-core/src/main/java/org/springframework/integration/channel/interceptor/WireTap.java

```diff
@@ -39,6 +40,7 @@
  * @author Gary Russell
  */
 @ManagedResource
+@IntegrationManagedResource
```

Comment:
Yes; I explained it in the commit comment (and added it as a comment here when I pushed).

>... There are a number of
standard beans (such as WireTap) that are @ManagedResources and
these would disappear for users that don't have an IMBE.

>That said, such beans previously disappear completely when there
is an IMBE so now they are now annotated with both so that
they are exported by at least/most one of the exporters.